YTIButtonRenderer.h

```diff
@@ -1,4 +1,12 @@
 #import <Foundation/Foundation.h>
+#import "YTICommand.h"
+#import "YTIFormattedString.h"
+#import "YTIIcon.h"
 
 @interface YTIButtonRenderer : NSObject
-@end
\ No newline at end of file
+@property(retain, nonatomic) YTICommand *command;
```

Comment:
```suggestion
@property (retain, nonatomic) YTICommand *command;
```
Maintain this format.